sympy/series/formal.py

```diff
@@ -1015,6 +1019,8 @@ def truncate(self, n=6):
         return self.polynomial(n) + Order(pt_xk, (x, x0))
 
     def _eval_term(self, pt):
+        if (self.args[0].is_polynomial()):
```

Comment:
Use `self.function` instead of `self.args[0]`.

Reply:
Done @ylemkimon